corehq/apps/events/forms.py

```diff
@@ -104,6 +111,38 @@ def __init__(self, *args, **kwargs):
         self.fields['expected_attendees'].choices = self.get_attendee_choices()
         self.fields['attendance_takers'].choices = self._get_possible_attendance_takers_ids()
 
+        self.fields['name'].disabled = not fields_should_be_available['name']
+        self.fields['start_date'].disabled = not fields_should_be_available['start_date']
+        self.fields['attendance_target'].disabled = not fields_should_be_available['attendance_target']
+        self.fields['expected_attendees'].disabled = not fields_should_be_available['expected_attendees']
+        self.fields['attendance_takers'].disabled = not fields_should_be_available['attendance_takers']
+
+    def determine_field_availability(self, event):
+        availability = {
+            'name': int(True),
+            'start_date': int(True),
+            'end_date': int(True),
+            'attendance_target': int(True),
+            'sameday_reg': int(True),
+            'tracking_option': int(True),
+            'expected_attendees': int(True),
+            'attendance_takers': int(True),
+        }
+
+        if event:
+            event_not_started = event.attendee_list_status == NOT_STARTED
+            event_in_progress = event.attendee_list_status == IN_PROGRESS
+            availability['name'] = int(event_not_started)
+            availability['start_date'] = int(event_not_started)
+            availability['end_date'] = int(event_not_started or event_in_progress)
+            availability['attendance_target'] = int(event_not_started)
+            availability['sameday_reg'] = int(event_not_started or event_in_progress)
+            availability['tracking_option'] = int(event_not_started)
+            availability['expected_attendees'] = int(event_not_started)
+            availability['attendance_takers'] = int(event_not_started)
```

Comment:
Should attendance takers be disabled for an event that has started? The ticket mentions that the disabled changes should not extend to attendance takers. :thinking:

Reply:
Ah, thanks for pointing that out. Totally forgot about that :sweat_smile: